db/version_set.cc

```diff
@@ -1986,12 +1988,14 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
     tracing_get_id = vset_->block_cache_tracer_->NextGetId();
   }
 
+  ReadOptions read_opts = read_options;
+  read_opts.rate_limiter_priority = Env::IO_USER;
   // Note: the old StackableDB-based BlobDB passes in
   // GetImplOptions::is_blob_index; for the integrated BlobDB implementation, we
   // need to provide it here.
   bool is_blob_index = false;
   bool* const is_blob_to_use = is_blob ? is_blob : &is_blob_index;
-  BlobFetcher blob_fetcher(this, read_options);
+  BlobFetcher blob_fetcher(this, read_opts);
```

Comment:
We should be using the user provided read_options.

Reply:
This was changed in a following commit.